documents/src/pages/elements/clock.md

```diff
@@ -234,3 +234,12 @@ ef-clock {
 <ef-clock analogue tick show-seconds am-pm></ef-clock>
 ```
 
+
+## Accessibility
+::a11y-intro::
+
+`ef-clock` is a set of number segments that contain hours, minutes, seconds, and time convention periods. It is assigned with `role="group"` and can include `aria-label` to describe the context of time.
```

Comment:
I do not think we need any of this text, as the clock works out of the box and does not require any additional work from application developers. @dtanp-rft , do you agree?

Reply:
Resolved by Alek PR.

Reply:
Yes the statement is wrong. I've revised this section and already push the changes.